homeassistant/components/ruuvitag_ble/strings.json

```
@@ -18,5 +18,24 @@
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "acceleration_total": {
+        "name": "Acceleration Total"
```

This should be sentence-cased:

```suggestion
        "name": "Acceleration total"
```

The "X" / "Y" / "Z" are variable names and correct.